pkg/controllers/collaset/synccontrol/sync_control.go

```diff
@@ -427,100 +628,113 @@ func (r *RealSyncControl) Update(
 	}
 	// 2. decide Pod update candidates
 	podToUpdate := decidePodToUpdate(cls, podUpdateInfos)
-
-	// 3. prepare Pods to begin PodOpsLifecycle
 	podCh := make(chan *PodUpdateInfo, len(podToUpdate))
-	for i, podInfo := range podToUpdate {
-		if podInfo.IsUpdatedRevision && !podInfo.PodDecorationChanged {
-			continue
-		}
-		if podopslifecycle.IsDuringOps(collasetutils.UpdateOpsLifecycleAdapter, podInfo) {
-			continue
-		}
-		podCh <- podToUpdate[i]
-	}
-
-	// 4. begin podOpsLifecycle parallel
 	updating := false
-	succCount, err := controllerutils.SlowStartBatch(len(podCh), controllerutils.SlowStartInitialBatchSize, false, func(_ int, err error) error {
-		podInfo := <-podCh
 
-		logger.V(1).Info("try to begin PodOpsLifecycle for updating Pod of CollaSet", "pod", commonutils.ObjectKeyString(podInfo.Pod))
-		if updated, err := podopslifecycle.Begin(r.client, collasetutils.UpdateOpsLifecycleAdapter, podInfo.Pod); err != nil {
-			return fmt.Errorf("fail to begin PodOpsLifecycle for updating Pod %s/%s: %s", podInfo.Namespace, podInfo.Name, err)
-		} else if updated {
-			// add an expectation for this pod update, before next reconciling
-			if err := collasetutils.ActiveExpectations.ExpectUpdate(cls, expectations.Pod, podInfo.Name, podInfo.ResourceVersion); err != nil {
-				return err
+	if cls.Spec.UpdateStrategy.PodUpdatePolicy != appsv1alpha1.CollaSetReplaceUpdatePodUpdateStrategyType {
```

Comment:
The IF condition here should be `PodInfo.isInReplacing == true` in case user changes the PodUpdateStrategy to other type during Pod replace updating.

And `PodInfo.isInReplacing` should be true when Pod has label `PodReplaceByReplaceUpdateLabelKey`.

Reply:
the case(from replaceUpdate to inPlaceUpdate) is handled by syncPods method.  the code here will not be changed